news/2024/2024-08-17-osu-lazer-updates-august-16-2024.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "osu!(lazer) Updates: August 16, 2024"
+date: 2024-08-17 20:00:00 +0000
+---
+
+The devs have been cooking, and a new batch of osu!(lazer) updates has just exited the oven.
+
+![](/wiki/shared/news/2024-08-17-osu-lazer-updates-august-16-2024/banner.jpg)
+
+With [COE](/wiki/Community/cavoe's_osu!_event) being over, it's unfortunately time to get back to our usual schedule. As always, you can check out a video digest of yesterday's update below!
```

Comment:
current wording makes it seem like the unfortunate thing is getting lazer updates Lol
```suggestion
With [COE](/wiki/Community/cavoe's_osu!_event) unfortunately being over, it's time to get back to our usual schedule. As always, you can check out a video digest of yesterday's update below!
```